cmd/libs/go2idl/generator/import_tracker.go

```diff
@@ -44,48 +49,110 @@ func NewImportTracker(types ...*types.Type) *ImportTracker {
 			// Add other forbidden keywords that also happen to be
 			// package names here.
 		},
+		lang: "golang",
 	}
 	tracker.AddTypes(types...)
 	return tracker
 }
 
+func NewProtobufImportTracker(local types.Name, types ...*types.Type) *ImportTracker {
```

Comment:
So my thinking about it is the following:
We are going to support protobuf-based API soon. That means, protobuf will be our "API-official" languange and in that case, I think it's fine to support proto in core go2idl.
We don't have any other API-official languages yet.

WDYT?